tests/DCPS/KeyTest/KeyMarshalling.cpp

```diff
@@ -31,6 +31,23 @@ void print_hex(void* d)
   printf("\n");
 }
 
+template<typename Type>
+void key_hash_test(const Type& value, SerializedSizeBound expected_bound = SerializedSizeBound())
+{
+  const SerializedSizeBound actual_bound =
+    MarshalTraits<Type>::key_only_serialized_size_bound(encoding);
+  std::cout << "  bound = " << actual_bound.to_string() << std::endl;
+  if (actual_bound != expected_bound) {
+    ACE_ERROR((LM_ERROR, "ERROR: expected bound to be %C, but it is %C\n",
+       expected_bound.to_string().c_str(), actual_bound.to_string().c_str()));
+    TEST_CHECK(false);
+  }
+
+  OpenDDS::RTPS::KeyHash_t hash;
+  OpenDDS::RTPS::marshal_key_hash(value, hash);
+  print_hex(hash.value);
```

Comment:
So I just simplified what the test was doing, but shouldn't it be checking that the result of the hash is right?

Reply:
I'm OK with this not being in scope.  Key hashes are not used.